### PR TITLE
petri net generator, workflow renderer

### PR DESCRIPTION
**Pull request by FiroozehDastur**
The branch includes the following features:
* XML workflow generator methods,
* render workflows and store them in PNG format.
* Added example scripts for some small cases

### CODE REVIEW

File: src/workflow_renderer.jl [hunk 1]
```diff
@@ -0,0 +1,205 @@
+
+# =============================================================================
+# Petri net viewer using GraphViz
+# =============================================================================
+"""
+    view_workflow(pnet::PetriNet, path::String)
+By default this method generates a png file after compiling the Petri net into an XML workflow and compiling the workflow.
```
Comment (mzeyen1985) [inline src/workflow_renderer.jl:7]: How is the output format selected by the user?

File: src/xml_generator.jl [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+julia> connect(pn, :in, p2)
+A Petri net with name "hello_julia", having 2 ports, 3 places, and 1 transitions.
+
+
+julia> connect(pn, :out, p3)
+A Petri net with name "hello_julia", having 3 ports, 3 places, and 1 transitions.
+
+
+julia> workflow_generator(pn)
+"An XML workflow "hello_julia" has been written to the location: /root/tmp."
```
Comment (mzeyen1985) [inline src/xml_generator.jl:232]: Maybe replace `/root/` with `/home/user/`

Comment (FiroozehDastur) [reply src/xml_generator.jl:232]: the message string is auto generated by Julia. But I can change the location to store the generated image. That way the message also changes.

File: src/workflow_renderer.jl (hunk 1 above)
Comment (FiroozehDastur) [reply src/workflow_renderer.jl:7]: I forgot to add that feature. Opps. Will work on it

File: README.md [hunk 3]
```diff
@@ -56,19 +59,52 @@ As a next step, it is advisible to create a script with your workflow configurat
...
-* Improve UI
-* Extend the interface 
+# ToDo...
+* Add examples for a cluster run.
+* Improve UI by adding documentation and examples.
+* Extend the interface to add more features.
 
 
 # Appendix
 The underlying workflow management system is called [GPI-Space](https://www.gpi-space.de/) which is a a task-based workflow management system for parallel applications developed at Fraunhofer ITWM by the CC-HPC group.
```
[the remark this answers is not in the document]
Comment (FiroozehDastur) [reply README.md:99]: will do

[the remark this answers is not in the document]
Comment (FiroozehDastur) [reply README.md]: done

File: src/workflow_renderer.jl (hunk 1 above)
Comment (FiroozehDastur) [reply src/workflow_renderer.jl]: added options for more than one file format. But default remains PNG

**Pull request merged**